### ISSUE TITLE
Auto add country prefix

### ISSUE THREAD

**Issue by yisraeldov** [64characters/Telephone#101]
Most of my contacts in my address book do not have the country code by their phone number if they are in my country. There should be an option to add a default country code if one is not found.

For example the the number 077-22-5555 should dial 97277225555 if my country is set to israel.

**Comment by poxin13** [comment]
Agree, this would be handy. Not all dials are from contacts, inputting 10 digit manually would be nice to auto add my country code.
